content/en/docs/k8s-operator/automatic.md

```diff
@@ -112,7 +112,68 @@ Coming Soon
 
 ### Java
 
-Coming Soon
+The following command will create a basic Instrumentation resource that is
```

Comment:
```suggestion
The following command will create a basic instrumentation resource that is
```

Reply:
`Instrumentation` is a proper noun here I think since it is the type of the customer k8s resource we are creating.

Reply:
Might it make sense for it to be in code font then?

Reply:
Or possibly, on the first occurrence, link to https://github.com/open-telemetry/opentelemetry-operator/blob/main/docs/api.md#instrumentation?

Reply:
@chalin the page links to the Instrumentation CRDs with its first occurrence here: https://github.com/open-telemetry/opentelemetry.io/blob/92e34768e7885423b0fd719dcbb77a8b08f53573/content/en/docs/k8s-operator/automatic.md?plain=1#L103.  Is that acceptable?

Reply:
Yes! Sorry, missed that.